appveyor.yml

```diff
@@ -17,6 +18,6 @@ test: off
 
 artifacts:
     - path: .\artifacts\logs\*.*
-    - path: .\artifacts\output\*-netstd.nupkg
+    - path: .\artifacts\output\*-netstd*.nupkg

```

Comment:
As well as the rebase, I had to amend the commit with this change.

Interestingly, GitHub did not invalidate the approval of the previous diff. :thinking: 

@FakeItEasy/owners I'd like to get your re-approval for this.